src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/global.yml

```diff
@@ -6,22 +6,22 @@ Transform: "AWS::Serverless-2016-10-31"
 Description: ADF CloudFormation Template (Global) for Deployment Account
 Parameters:
   ADFVersion:
-    Type : "AWS::SSM::Parameter::Value<String>"
+    Type: "AWS::SSM::Parameter::Value<String>"
```

Comment:
Could you please undo the linting changes made to this file?
Otherwise it will cause a lot of conflicts with another PR that is in progress.